server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
+  user_id: { type: Number, ref: 'User', required: true },
+  application_id: { type: Number, ref: 'Job_Application', required: true },
+  interview_date: { type: Date },
+  interview_time: { type: String },
+  interview_location: { type: String },
+  interviewer_name: { type: String },
```

Comment:
We'll probably need to support multiple interviewers. Also, maybe the interviewers should be Network(ing)Contacts?

Reply:
Changed interviewer_name to networkingContact, and added a ref to that model

Reply:
Nice. I think we may need to tweak it though.

Maybe from:
NetworkingContact: [{ type: String, ref: 'NetworkingContact'}],

to:
networkingContactIds: [{ type: mongoose.Schema.Types.ObjectId, ref: 'NetworkingContact'}],